spec/classes/puppet_server_spec.rb

```diff
@@ -1,181 +1,754 @@
 require 'spec_helper'
 
-describe 'puppet::server' do
+describe 'puppet' do
   on_os_under_test.each do |os, facts|
-    next if unsupported_puppetmaster_osfamily(facts[:osfamily])
-    context "on #{os}" do
-      if facts[:osfamily] == 'FreeBSD'
-        ssldir = '/var/puppet/ssl'
-      else
-        ssldir = '/etc/puppetlabs/puppet/ssl'
+    context "on #{os}", unless: unsupported_puppetmaster_osfamily(facts[:osfamily]) do
+      # TODO: only used in 1 test so not before each?
```

Comment:
still TODO? ;)

Reply:
It was, but moved it to a `before :context` in that particular case.